src/java.base/share/classes/jdk/internal/vm/annotation/JvmtiMountTransition.java

```diff
@@ -30,9 +30,12 @@
 /**
  * A method is annotated as "jvmti mount transition" if it starts
  * or ends virtual thread mount state transition (VTMS transition).
+ * The Continuation yield and yield0 frames normally are in VTMS transition
+ * but can be found out of transition in an unmounted virtual thread.
+ * This inconsistency is the reason why they also need this annotation.
```

Comment:
In ChangesCurrentThread, Hidden and the other annotations, the description explains where they must be used. The JvmtiMountTransition description isn't very clear as it's explain why the Continuation yield methods need the annotation. It's okay if the description is re-worked in some future PR, my comment is just that the current description is very confusing.